src/utils/db_manager.py

```
@@ -1,49 +1,170 @@
+                    City TEXT,
+                    TotalScore INTEGER,
+                    Tier TEXT,
+                    EstimatedValue TEXT,
+                    Opportunities TEXT,
+                    CreatedDate TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+                )
+            ''')
+            
+            conn.commit()
+        except sqlite3.Error as e:
+            logging.error(f"Database initialization failed: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)